### DISCUSSION TITLE
Live Timing Width Varies

### DISCUSSION THREAD

**Discussion by jaa101** [f1multiviewer/issue-tracker#143] [Feature requests] [answered]
I size the Live Timing window at the start of the race with the narrowest width so that all the stats are visible (in a single column and the with messages popped out).  I then arrange the various video windows to fill the rest of my screen.  As soon as some timing data appear on the Live Timing its graphics are pushed wider so that the rightmost part (tyre info) become hidden.  I then have to resize all my windows.

Could we change the Live Timing graphics so that all the elements are fixed width, independent of data (or lack thereof) so that the layout doesn't change during a session.  Thanks.

**Comment by f1multiviewer** [comment, marked as answer]
This will be mitigated in most cases in the upcoming release!
